apps/api-evidence/src/app/api/users.py

```diff
@@ -52,6 +52,27 @@ def create_user(
     return user
 
 
+@router.put('/{user_id}', summary='Update User (Admin)', response_model=schemas.User)
+def update_user(
+    *,
+    db: Session = Depends(deps.get_db),
+    user_id: int,
+    user_in: schemas.UserUpdate,
+    current_user: models.User = Depends(deps.get_current_active_superuser),
+) -> Any:
+    """
+    Update a user.
+    """
+    user = crud.user.get(db, id=user_id)
+    if not user:
+        raise HTTPException(
+            status_code=404,
+            detail='The user does not exist',
```

Comment:
I think for `404` we don't need a detail/message, it should be implied that the requested resource does not exist.